modules/alloy/src/test/ts/browser/behaviour/TooltippingTest.ts

```diff
@@ -53,10 +42,10 @@ UnitTest.asynctest('Tooltipping Behaviour', (success, failure) => {
             GuiFactory.text(`${name}-tooltip`)
           ]
         }),
-        Focusing.config({ }),
```

Comment:
Normally eslint prefers to keep the space inside between brackets

Reply:
This and all the other lint issues makes me think the linter isn't doing much (if anything) on alloy 🤔